docs/examples/plotting/auto_color_scale.ipynb

```diff
@@ -482,20 +481,27 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Because of the possible risks due to auto color scaling it is deactivated by default. Please remember to change the colors marking the outliers in case you should use a color map the includes white and gray."
+    "Because of the possible risks due to auto color scaling it is deactivated by default. Please remember to change the colors marking the outliers in case you should use a color map the includes white and grey."
```

Comment:
I think (but I am not 100% sure that) at some point we decided to stick to American English, because our British English was not good enough ;-).

Reply:
So I just checked for the 15mins to QCoDeS. This is mostly US English. We should probably take a decision all together.

Reply:
I think somewhere in the same notebook ı saw another "grey" so I just changed this one for coherency. From the only documentation point of view, US English is the accepted norm. Let me check this notebook very fast again.